win32.c

```diff
@@ -9,6 +9,8 @@
 #include <windows.h>
 #include <mmsystem.h>
 
+#pragma warning (disable : 4996)
```

Comment:
Why should a warning be silenced like this? What is this warning?

Reply:
msvc 2019 complains about insecure or obsolete functions that the enet src code is using and disabling the warning C4996 allows you to compile the program with such acknowledgment. It has no negative effects, I could flawlessly use ENet after this patch.

Reply:
It's at least worth a comment.

Reply:
I'm not sure warnings should be silenced like this, can't you disable them in your custom build settings?

Reply:
Just define _CRT_SECURE_NO_WARNINGS for the Win32 case.